### PR TITLE
Finish home page

### PR DESCRIPTION
**Pull request by juanmanuelbyc**
Create home page and about page for capstone 1 project

Project requirements

Personalize the content of your page. Choose a topic that is different than the one in the original design.

Follow design guidlines, including:

Colors.
Typography: font face, size and weight.
Layout: composition and space between elements.
The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

Build these 2 pages:
The home page.
The about page.

Each of these pages should have versions for 2 different screen sizes:
Mobile: up to 768px wide.
Desktop: 768px or wider.

Interactions
Links
The home page should have a link in the menu to the about page.
The logo in the header links to the home page.
Mobile menu
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
The mobile menu should have a close (X) button that closes the menu.
Dynamic page

The section "Featured speakers" should be created dynamically in JavaScript.
You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+## 📝 License
+
+This project doesn't have or require any license.
+
+## Preview
+
+This is how this project looks like
+
+[here!](images/screenshot.jpg)
```
Comment (Ismailco) [inline README.md:59]: - [ ] This link is not working kindly fix it
- [ ] Kindly include a link to a video presentation about your app, the video should have these requirements:
  - The student recorded a video presenting the features of the project.
  -  The maximum length of the video is 5 minutes.
  -  The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  -  The video contains a demo of the project, showing the pages in different screen sizes.
  -  The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+
+## Online version
+
+No Online Version
```
Comment (Ismailco) [inline README.md]: - [ ] Kindly add a demo link to the online website hosted with the Github Pages, this is a requirement, and also users can see the live version of your site

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+
+- GitHub: [@juanmanuelbyc](https://github.com/juanmanuelbyc)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/juanmanuelbyc/Capstone1/issues).
+
```
Comment (Ismailco) [inline README.md:45]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+          </ul>
+        </nav>
+      </div>
+      <div class="header-div header-bottom flex">
+        <div class="logo">
+          <a href="index.html"><img alt="moto logo" class="my-logo" src="images/moto-logo.png"></a>
+        </div>
+        <nav class="menu-container flex align-items-lg-center gap-5">
+          <ul class="menu-list bottom-header-list flex gap-5 list-unstyled">
+            <li><a href="about.html" class="btm-header-option" aria-label="About">About</a></li>
```
Comment (Ismailco) [inline about.html]: - [ ] It should be possible to navigate between the pages, kindly add a link to the `Home` page in this `About` page so the user can navigate easily between pages, you can change the About link in the nav bar to Home
e.g `<li><a href="index.html" class="btm-header-option" aria-label="About">Home</a></li>`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+            <li><a href="#" class="btm-header-option">Join</a></li>
+            <li><a href="#" class="btm-header-option">Sponsor</a></li>
+            <li><a href="#" class="btm-header-option">News</a></li>   
+          </ul>
+          <div class="cc-link flex justify-content-center">
+            <a>Bikers Club</a>
+          </div>
+        </nav> 
+      </div>
+    </header>
```
Comment (Ismailco) [inline index.html]: - [ ] The pages use the exact same colors as the original design, the top bar should be darker, the color of the links in the navigation bar should be darker, and when hover should be changed to `#ec5242`

|Your design|CC design|
|-|-|
|Screenshot_20220727_224940|Screenshot_20220727_225130|

File: index.html [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+      </div>
+
+
+    <!-- Main section -->
+
+      <section class="intro flex flex-column section">
+        <div class="menu-icon-container">
+          <img class="menu-icon" src="images/icons/menu.png" alt="menu-button">
+        </div>
+        <div class="intro-content-container flex flex-column content-container">
```
Comment (Ismailco) [inline index.html:82]: - [ ] [OPTIONAL] You could use the `<article>` tag inside the section for parts that are stand-alone sections or have the same content so your code can use even more semantic HTML

File: speakers.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+   {
+      id: 6,
+      name: 'HD XR750 Cut away bike',
+      position: 'This bike illustrates the most dominant production racer ever built',
+      activity: 'Today’s XR750 racers are very similar to the first “alloy” versions from 1972.',
+      imageUrl: 'images/harley750.jpg',
+      }
+]
+
+document.querySelector('.speakers-container').innerHTML = speakers.map((speaker) =>
```
Comment (Ismailco) [inline speakers.js:46]: - [ ] [OPTIONAL] You can add something like `window.onload = () => { Your code }` so the page populate on window load

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @juanmanuelbyc, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success
Hey @juanmanuelbyc :wave:

Great work on implementing the changes requested. Your project is complete. There is nothing else to say about your project. :+1:
Go ahead and merge it. :muscle:

## Highlights :1st_place_medal:
:heavy_check_mark: Following GitFlow
:heavy_check_mark: Following Linters
:heavy_check_mark: Following the Design
:heavy_check_mark: Descriptive PR summary

### [Optional]
**[Optional]**
- [ ] Please make your PR title more descriptive such as **Final Version Capstone** :blush:
- [ ] Please make your branch name more descriptive such as **feature/capstone-project** :+1:

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**